### DISCUSSION TITLE
The new one is nice, but getting used to the old How can I revert to the old code view?

### DISCUSSION THREAD

**Discussion by awe22** [community/community#55425] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Big kudos to GitHub for forcing the new code search and code review features. These new features provide more flexibility and stronger search capabilities, making it easier for developers to find the code they need.

However, I understand that some people may get used to the old way of code search, and that the new code search may not be fully functional and some users may find it inconvenient. In response to this problem, I suggest that GitHub can consider the following measures:
Provide options: To ensure a smooth transition to the new code search method for all users during the transition, GitHub could consider providing options to allow users to choose their preferred search method.
In short, I think GitHub's mandatory opening of new code search and code viewing functions is a very good move, but we also need to pay attention to user feedback and feelings to ensure that these new features really bring convenience to users.
Finally, there are three questions that are not clear now
Does this mean that all github users are forced to use the latest version?
Could you please restore our ability to get the switch to the old data?
The button in the previous function preview has disappeared. Is there a new solution?

**Comment by uyqadri** [comment]
It would be good if recent indexed search code option enabled again

**Comment by le2Ks** [comment, marked as answer]
They said the new system is not compatible with time sorted results. That, and many other bad things. In short, yes we are stuck with this useless version. And apparently they are not interested in putting back the old one. 
This is interesting however that even my youngest daughter of 12 years old noticed how bad it was, and started to talk about it yesterday during dinner.
Yet, a full team of "pro" developers in Github who build this system, cannot understand where the problem is.

This is really sad.. not even mad anymore, just really sad to see how it became. This, my friends, is the beginning of the end of Github.
